### ISSUE TITLE
Custom Media Queries (@custom-media)

### ISSUE THREAD

**Issue by kbjr** [i-akhmadullin/Sublime-CSS3#24]
The spec for custom media queries is can be found here: http://dev.w3.org/csswg/mediaqueries/#custom-mq

Per the above link, the syntax is:

```
@custom-media = @custom-media <extension-name> <media-query-list> ;
```

``` css
@custom-media --narrow-window (max-width: 30em);

@media (--narrow-window) {
  /* narrow window styles */
}
@media (--narrow-window) and (script) {
  /* special styles for when script is allowed */
}
```

**Comment by i-akhmadullin** [comment]
Have they been implemented in any browser ?

**Comment by kbjr** [comment]
Not that I'm aware of, but I haven't tested. I'm using them with [myth](http://myth.io), so they're being compiled away.

**Comment by i-akhmadullin** [comment]
Fixed in 1.0.10 (b1ba73806d67871d748be4264ea1b9f76ecc53c3)
![custom-media queries highlight](https://cloud.githubusercontent.com/assets/1042267/3590461/55783278-0c59-11e4-8019-107236e2d269.png)
There is no highlighting for alias name and probably never will be because it's not possible with regexes as far as i know.
